nix/profile.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, writeScriptBin, gnutar, kore-exec-prof, kore-exec-infotable, hp2pretty, hs-speedscope, eventlog2html }:
+
+writeScriptBin "profile" ''
```

Comment:
Somewhat bike-shedding... but could we rename this file and its output to something like `run-profile` or `profiling-run`?
I was reading `nix/profile.nix` and thought "an entirely new environment profile for nix?" :)

Reply:
sure, i'll change to `run-profiling` (to split the difference :P)